_episodes/02-data-management.md

```diff
@@ -149,3 +178,5 @@ names(data)
 > > B.  38
 > {: .solution}
 {: .challenge}
+
+[data]: {{ site.baseurl }}/data/gapminder_all.csv
```

Comment:
+1 to putting all the URLs at the bottom like this - we should do that for all our lessons.